src/ess/sans/workflow.py

```diff
@@ -52,27 +52,31 @@ def with_pixel_mask_filenames(
     return workflow
 
 
-def with_banks(workflow: sciline.Pipeline, banks: Iterable[str]) -> sciline.Pipeline:
+def with_banks(
+    workflow: sciline.Pipeline,
+    banks: Iterable[str],
+    index: Iterable[Hashable] | None = None,
+) -> sciline.Pipeline:
     """
     Return modified workflow with bank names set.
 
+    Since banks typically have different Q-resolution the I(Q) of banks are not merged.
```

Comment:
So does this mean that when we merge from multiple runs, we need to check that the detector banks were in the same position? The rear panel on Loki can move quite a lot...

Reply:
... and we may also need to split runs, if the sample env params are different. There will be more work to do here in the future, I think.